ansible/library/port_alias.py

```diff
@@ -234,10 +234,11 @@ def main():
                                            'asic_if_names': asic_if_names,
                                            'sysports': sysports})
            return
+
         allmap = SonicPortAliasMap(m_args['hwsku'])
-        start_switchid = 0
-        if  'start_switchid' in m_args and m_args['start_switchid'] != None:
-           start_switchid = int(m_args['start_switchid'])
+        switchids = [0]
+        if 'switchids' in m_args and m_args['switchids'] != None:
```

Comment:
can there be an assert for len(switchids) == num_asic ?

Reply:
There was an issue in port_alias which would fail if switchids was not passed. I have addressed this in commit 7daf2e0 to be backward compatible. When port_alias is called from tests, they don't need to pass swtichids.